waltid-applications/waltid-cli/src/jvmTest/kotlin/id/walt/cli/commands/WaltIdDidResolveCmdTest.kt

```diff
@@ -39,112 +39,112 @@ class WaltIdDidResolveCmdTest {
 
     @Test
     @Ignore
-    fun `should resolve uniresolver P-521 did-key example`() {
+    fun `should resolve uniresolver P-521 did|key example`() {
```

Comment:
pipe char is an invalid char on windows and throws 'invalid path error'
this is also the reason it was changed to '-'

Reply:
merge weirdness, rebased with current main